committee.html

```diff
@@ -178,10 +178,10 @@ <h1 class="darkcolor">THE COMMITTEE</h1>
                             <div class="card shadow-sm h-100">
                               <svg class="bd-placeholder-img card-img-top" viewBox="0 0 100 100" role="img" preserveAspectRatio="xMidYMid meet" focusable="false"><image href="img/team/shantanu.jpeg" width="100%" height="100%"></svg>
                               <div class="card-body">
-                                <p class="card-text custom-font bold-font darkcolor" style="font-size:160%;">Shantanu Kadam</p>
-                                <p class="text-muted custom-font darkcolor">Physics</p>
+                                <p class="card-text custom-font bold-font darkcolor" style="font-size:160%;  padding-top: 10px !important;">Shantanu Kadam</p>
+                                <p class="text-muted custom-font darkcolor medium-font">Physics, Environmental & Economic Policy</p>
```

Comment:
Would you mind changing the & to an "and" in Environmental "&" Economic Policy? Keeps things consistent.